.travis.yml

```diff
@@ -24,3 +24,9 @@ jobs:
     script:
     - ./build all-test netlrts-linux-arm8 -j3 --with-production --enable-error-checking --enable-lbuserdata -g -Werror=vla || exit 1
     - make -C netlrts-linux-arm8/tmp test TESTOPTS="++local"
+  - os: linux
+    arch: ppc64le
+    name: "NetLRTS-Power"
+    script:
+    - ./build all-test netlrts-linux-ppc64le -j3 --with-production --enable-error-checking --enable-lbuserdata -g -Werror=vla || exit 1
+    - make -C netlrts-linux-ppc64le/tmp test TESTOPTS="++local"
```

Comment:
```suggestion
    - ./buildold all-test pamilrts-linux-ppc64le gcc -j3 --with-production --enable-error-checking --enable-lbuserdata -g -Werror=vla || exit 1
    - make -C pamilrts-linux-ppc64le-gcc/tmp test TESTOPTS="++local"
```

Reply:
@rbuch Could we still test this with buidold? I think we currently have no pami* test at all.

Reply:
I think that configuration will still end up with xlC_r in CMK_NATIVE_CXX, which is not present on these machines.

Reply:
In addition to not having the XL compilers installed, it's also unlikely that these machines have PAMI installed.